src/Subscriptions/Broadcasters/EchoBroadcaster.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Nuwave\Lighthouse\Subscriptions\Broadcasters;
+
+use Illuminate\Broadcasting\BroadcastManager;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Arr;
+use Nuwave\Lighthouse\Subscriptions\Contracts\Broadcaster;
+use Nuwave\Lighthouse\Subscriptions\Events\EchoSubscriptionEvent;
+use Nuwave\Lighthouse\Subscriptions\Subscriber;
+use Symfony\Component\HttpFoundation\Response;
+
+class EchoBroadcaster implements Broadcaster
+{
+    /**
+     * @var BroadcastManager
+     */
+    protected $broadcaster;
+
+    public function __construct(BroadcastManager $broadcaster)
+    {
+        $this->broadcaster = $broadcaster;
+    }
+
+    /**
+     * @param array<string, mixed> $data
```

Comment:
```suggestion
     * @param  array<string, mixed>  $data
```

Laravel coding style, too bad the autoformatter does not catch it.